Misc updates for v31.0.4

tcprescott opened the pull request:
Add a fairly comprehensive list of changes made for v31.0.4 so far.
Add rudimentary Enemizer support to the customizer.
Allow customizer users to disable prize pack customization (useful if the player wants to use standard v31 prize pack generation).
A couple of site copy updates

sporchia commented on resources/js/components/Customizer/PrizePacks.vue:
```diff
@@ -123,23 +138,43 @@
   components: {
-    Select
+    Toggle: Toggle,
+    Select: Select
```
this should be able to be written as:
```
Toggle,
Select
```

sporchia commented on resources/js/components/Customizer/PrizePacks.vue:
```diff
@@ -123,23 +138,43 @@
+    }
+  },
+  created() {
+    this.customPrizePacks = this.$store.state.context.settings["customPrizePacks"]
```
I believe we generally avoid local state and use computed's introspecting on the store

tcprescott commented:
Removing prize pack customization stuff for now.

tcprescott commented:
This last commit should contain some Crowd Control item pool fixes.  It is ready to merge, Veetorp.

tcprescott commented:
Made some additional updates to the Crowd Control item pool
Added a "/api/daily" endpoint that returns the hash and date of the current daily as JSON
Fixed an issue that was made it impossible to manually place Uncle's item when using randomized swords in standard state.
